.ci/pytorch/build.sh

```diff
@@ -6,6 +6,11 @@ set -ex
 # (This is set by default in the Docker images we build, so you don't
 # need to set it yourself.
 
+if [[ "$BUILD_ENVIRONMENT" == linux-jammy-py3.8-gcc11* ]] ; then
+  echo "Sometimes in kernel 5+ anaconda decides to use its own libstdc++ instead of the system one. As we've always been relying on the system one, let's make sure it's used."
```

Comment:
Please search for existing issue or create a new issue for us in ``pytorch/pytorch`` with description of the error you are seeing.
Also worthwhile checking the anaconda issues here:
https://github.com/ContinuumIO/anaconda-issues i

Reply:
Opened a issue with very clear description of the origin of the problem https://github.com/pytorch/pytorch/issues/121796

TLDR: no, the problem is not related to ARC, and it does not necessarily happens on ARC runners. So doing this only when running on ARC won't help. I tried it in multiple ways :(

Reply:
Please aldo list this issue in the comments. We probably don't need to use echo here
Use something like this:
```
# Forces build to rely on system libstdc++ rather then anaconda provided one see https://github.com/pytorch/pytorch/issues/121796
```